heading_control_server_node:
  str_adj: 50
  hdg_p: 10.0
  hdg_i: 1.0
  hdg_d: 0.0
  skid_adj: 100
  skid_p: 1.0
  skid_i: 0.0
  skid_d: 0.0
  vel_setpoint: 0.5 #m/s
  vel_p: 15.0
  vel_i: 10.0
  vel_d: 1.0
